extension/src/main/webapp/WEB-INF/conf/gamification/global/completion-configuration.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
```

Comment:
Should be https://www.exoplatform.com/xml/ns/kernel_1_3.xsd

Reply:
Done